assets/sass/components/analytics-4/_googlesitekit-analytics-setup-module.scss

```diff
@@ -46,34 +46,40 @@
 				margin-bottom: 0;
 			}
 
-			.googlesitekit-analytics-enable-enhanced-measurement__already-enabled-label {
-				font-size: $fs-body-sm;
-				letter-spacing: $ls-xs;
-			}
-
-			.googlesitekit-module-settings-group__helper-text {
-				color: $c-surfaces-on-surface-variant;
-				font-size: $fs-body-sm;
-				letter-spacing: $ls-xs;
-				margin: 4px 0;
-			}
-
 			.googlesitekit-plugin-conversion-tracking-notice {
 				color: $c-surfaces-on-surface;
 				font-size: $fs-body-sm;
 				letter-spacing: $ls-xs;
 				margin: 23px 0 0;
 			}
 
-			.googlesitekit-analytics-enable-enhanced-measurement {
+			// Ensure that these styles are only applied within Analytics setup.
+			.googlesitekit-analytics-setup__form {
+
+				.googlesitekit-analytics-enable-enhanced-measurement {
+
+					.googlesitekit-module-settings-group__helper-text {
```

Comment:
This nesting ensures that these styles are only applied to the setup screen but not the settings screen eventually when the feature flag is removed.